src/js/components/Articles/ArticleReader.js

```diff
@@ -0,0 +1,192 @@
+import React, { Component } from "react";
+import Navbar from "../Navbar/Navbar";
+import { createGlobalStyle } from "styled-components";
+import { ReactReader } from "./modules";
+import { NavLink } from "react-router-dom";
+import "./ArticleButtons.scss";
+import Share from "./Share";
+import "../Fragments/Navbuttons.scss";
+
+import {
+  Container,
+  ReaderContainer,
+  FontSizeButton,
+  ExitFullScreenButton
+} from "./Components";
+
+const storage = global.localStorage || null;
+
+const GlobalStyle = createGlobalStyle`
+  * {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+    margin: 0;
```

Comment:
You can inherit the margin and padding from the body since body will have an higher priority

Reply:
I have updated it to its own .scss file with margin and padding removed